agera/src/test/java/com/google/android/agera/ObservablesTest.java

```diff
@@ -325,6 +325,32 @@ public void shouldUpdatePerMillisecondObservable() {
   }
 
   @Test
+  public void shouldUpdateCompositeOfPerMillisecondObservable() {
+    final long expectedDelayedTime = scheduler.getCurrentTime() + FILTER_TIME;
+    updatable.addToObservable(compositeObservable(perMillisecondObservable(
+        FILTER_TIME, updateDispatcher), updateDispatcher()));
+
+    updateDispatcher.update();
```

Comment:
shouldn't it be something like:
`updateDispatcher.update();
idleMainLooper(FILTER_TIME / 2);
updateDispatcher.update();
updateDispatcher.update();
idleMainLooper(FILTER_TIME / 2);
assertThat(updatable, wasUpdatedOnce());
`